pkg/util/exec/exec.go

```diff
@@ -113,7 +113,7 @@ func logOutput(name string, stdout, stderr io.ReadCloser) {
 }
 
 func runCommandWithOutput(actionName string, cmd *exec.Cmd) (string, error) {
-	output, err := cmd.Output()
+	output, err := cmd.CombinedOutput()
```

Comment:
do you need to collect stderr? i wonder if this will have side effects for other commands that have been ignoring stderr